client/lib/signup/step-actions.js

```diff
@@ -167,6 +168,16 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 	// flowName isn't always passed in
 	const flowToCheck = flowName || lastKnownFlow;
 
+	if (
+		flowToCheck === 'onboarding' &&
+		newSiteParams.options.theme !== defaultSiteTypeTheme &&
```

Comment:
@p-jackson: is the goal here to make sure that if the theme is Maywood, we use the vertical annotation instead of the theme annotation? If so, I don't think it's necessary - Maywood actually is a pretty great restaurant theme by default.

Reply:
Not specifically Maywood, the purpose is to test if we're using a vertical-suggested theme. If they choose Restaurants then the preview would have shown them Rockfield. If Maywood is selected then it musn't have been a vertical-suggested theme.

The template-first theme spreadsheet _does_ say Maywood's for restaurants, but suggests we use Rockfield.